sklearn/utils/tests/test_validation.py

```diff
@@ -187,6 +187,19 @@ def test_check_array_force_all_finiteinvalid(value, force_all_finite,
                     accept_sparse=True)
 
 
+def test_check_array_force_all_finite_object():
+    X = np.array([['a', 'b', np.nan]], dtype=object).T
+
+    X_checked = check_array(X, dtype=None, force_all_finite='allow-nan')
+    #assert_allclose_dense_sparse(X, X_checked)
```

Comment:
why is this commented?

Reply:
PEP8 won't like the # without a space

Reply:
Sorry, should have added a comment. `assert_allclose_dense_sparse` doesn't handle object dtype data, so if we want to check that `X` and `X_checked` are still the same, we need to write a custom assert method.

Reply:
In this case, isn't `assert X is X_checked` possible?